src/app/mainwin.cpp

```diff
@@ -144,7 +144,11 @@ int CALLBACK WinMain( HINSTANCE hInstance, HINSTANCE hPrevInstance, LPSTR lpCmdL
 
     wchar_t *path = wcsdup( _wgetenv( L"PATH" ) );
 
+#ifdef _UCRT
```

Comment:
According to https://learn.microsoft.com/en-us/cpp/c-runtime-library/reference/strtok-strtok-l-wcstok-wcstok-l-mbstok-mbstok-l?view=msvc-170, the 2 parameter wcstok() is non standard and there's the standard 3 parameters wcstok(), which is also the C++ std::wcstok (https://en.cppreference.com/w/cpp/string/wide/wcstok)
So you could just remove the #ifdef and use unconditionally the 3 parameter wcstok()

Reply:
It won't build with mingw-w64/msvcrt

Reply:
> It won't build with mingw-w64/msvcrt

and what if you use ``std::wcstok`` instead ? (with ``#include <cwchar>``)

Reply:
I don't know!

Reply:
> I don't know!

just try :-)

Reply:
I won't.

Reply:
I hate when people start suggesting things they don't know if they work or not, just to block fixing a bug!
Try it yourself and see if It works or not!